src/web/modules/gateway_accounts/detail.njk

```diff
@@ -1,12 +1,19 @@
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 {% from "common/json.njk" import json %}
 {% extends "layout/layout.njk" %}
 
 {% set isTestData = not account.live %}
 
 {% block main %}
+  {% if currentCredential.state  === 'CREATED' %}
+    {{ govukNotificationBanner({
+      text: 'This account needs to be configured with PSP credentials.'
```

Comment:
`PSP credentails` is only for Worldpay. For Stripe service need to complete onboarding. 

```suggestion
      text: 'This account needs to be configured with PSP credentials or complete onboarding.'
```

Reply:
Good spot - have updated it.